figures/scripts/fig4-tp53-telomerase-panels.R

```diff
@@ -613,3 +635,49 @@ forest_panels <- cowplot::plot_grid(forest_plot, labels_panel, nrow = 1, rel_wid
 # Export plot
 ggsave(survival_plot_pdf, forest_panels, width = 11, height = 3.5)
 
+
+## Export CSVs for Zenodo upload ------------------------------
+
+# Panel 4A: ROC curve
+# no sample information so no arranging is needed
+readr::write_csv(roc_df, fig4a_csv)
+
+# Panel 4B: TP53 scores violin plots
+tp53_scores_data %>%
+  # reorder columns so sample id first, and rename it to match what the metadata will have
+  dplyr::select(Kids_First_Biospecimen_ID = Kids_First_Biospecimen_ID_RNA, everything()) %>%
+  # arrange on sample
+  dplyr::arrange(Kids_First_Biospecimen_ID) %>%
+  # remove \n from tp53_altered so that CSV is properly formatted
+  dplyr::mutate(tp53_altered = stringr::str_replace(tp53_altered, "\n", " ")) %>%
```

Comment:
Yup, this is also just separating an (N=##) line, so let's get rid of that
```suggestion
  dplyr::mutate(tp53_altered = stringr::str_replace(tp53_altered, "\n.+", "")) %>%
```